cuegui/tests/CueJobMonitorTree_tests.py

```diff
@@ -0,0 +1,65 @@
+#  Copyright (c) OpenCue Project Authors
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+
+import mock
+import unittest
+
+import PySide2.QtCore
+import PySide2.QtGui
+import PySide2.QtWidgets
+
+import cuegui.CueJobMonitorTree
+import cuegui.plugins.MonitorCuePlugin
+import cuegui.Style
+import opencue.compiled_proto.job_pb2
+import opencue.compiled_proto.show_pb2
+import opencue.wrappers.show
+
+from . import test_utils
+
+
+@mock.patch('opencue.cuebot.Cuebot.getStub', new=mock.Mock())
+class CueJobMonitorTreeTests(unittest.TestCase):
+
+    @mock.patch('opencue.cuebot.Cuebot.getStub')
+    def setUp(self, get_stub_mock):
+        test_utils.createApplication()
+        PySide2.QtGui.qApp.settings = PySide2.QtCore.QSettings()
+        cuegui.Style.init()
+
+        show_name = 'arbitrary-show-name'
+        job_name = 'arbitrary-job-name'
```

Comment:
Slight preference: It looks like job_name is only used in the `ShowGetJobWhiteboardResponse` call which passes it as a list. Can we just define `jobs` here?

Reply:
Done.